docs/maintain/maintain-guides-polkadot-opengov.md

```diff
@@ -261,6 +260,45 @@ automatically populated. The enactment delay can be specified either as a block
 specific number of blocks after the referendum is approved. The deposit for this proposal will be
 locked for the referendum duration.
 
+### Submitting a Referendum on the Whitelisted Caller Track
+
+Let's consider increasing the active validator count. You could
```

Comment:
<img width="841" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/83986b8a-23d5-4a7d-a187-7dc6ea088871">

The example call sets the number of validators participating in the parachain consensus. Modifying the text to reflect that.


```suggestion
Let's consider increasing the number of validators participating in parachain consensus. You could
```